yarn-project/noir-protocol-circuits/src/crates/private-kernel-lib/src/private_kernel_init.nr

```diff
@@ -434,8 +434,8 @@ mod tests {
         let mut builder = PrivateKernelInitInputsBuilder::new();
 
         // Set the leaf index of the contract leaf to a wrong value (the correct value + 1).
-        let leaf_index = builder.private_call.contract_leaf_membership_witness.leaf_index;
-        builder.private_call.contract_leaf_membership_witness.leaf_index = leaf_index + 1;
+        // let leaf_index = builder.private_call.contract_leaf_membership_witness.leaf_index;
+        // builder.private_call.contract_leaf_membership_witness.leaf_index = leaf_index + 1;
 
         builder.failed();
```

Comment:
Just raising that this test is disabled right now. (the `#[test]` annotation is commented out).

At first I was confused why the test was expected to fail when it didn't deviate from the default setup and I was about to ask if the default set of contract/function call fixture was set up to fail by default, but then I noticed the test was disabled.

(same for the next test down)

Reply:
Yep! Still got to re-enable those tests. Will do in an upcoming PR.

Reply:
Fixed in https://github.com/AztecProtocol/aztec-packages/pull/4358